src/main/kotlin/water/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,23 @@
+package water.racingcar.domain
+
+class Car(val name: String) {
+    var movedValue: Int = 0
+        private set
+
+    fun move(randomNumber: Int) {
+        if (randomNumber >= 4)
+            movedValue++
+    }
+
+    override fun toString(): String {
+        val output = StringBuilder()
+        var moved = movedValue
+
+        output.append("$name : ")
+        while (moved > 0) {
+            output.append("-")
+            moved--
+        }
```

Comment:
```suggestion
        repeat(movedValue) {
            output.append("-")
        }
```
이렇게 바꾸면 moved를 따로 선언 안 해줘도 원하는 결과를 얻을 수 있을 거 같아요ㅎㅎ

Reply:
훨씬 좋은 방법이 있었군요!! 반영 후 커밋하겠습니다 👍